liquidity_pool/src/lib.rs

```diff
@@ -103,7 +103,7 @@ fn burn_shares(e: &Env, amount: BigInt) {
 
     TokenClient::new(&e, share_contract_id).burn(
         &Signature::Contract,
-        &BigInt::from_u32(&e, 0),
+        &BigInt::zero(&e),
```

Comment:
You can also use `bigint!(&e, 0)`.

Reply:
Do you have a preference? `zero` feels slightly safer.

Reply:
No pref for zero, bigint! will work for a wider variety of literals that from_ won't. But it doesn't matter for this.